extra-dev/packages/coq-hott/coq-hott.8.13.dev/opam

```diff
@@ -20,5 +18,5 @@ dev-repo: "git+https://github.com/HoTT/HoTT.git"
 synopsis: "The Homotopy Type Theory library"
```

Comment:
Please add a description explaining the options for using this package, both the coqc command line syntax and that it is recommended to use a _CoqPlatform file with contents:
```
-arg -noinit
-arg -indices-matter
```